packages/node-sdk/src/types.ts

```diff
@@ -190,6 +190,56 @@ export type FeatureOverride =
     })
   | boolean;
 
+/**
+ * Describes a feature definition.
+ */
+export type FeatureDefinition = {
+  /**
+   * The key of the feature.
+   */
+  key: string;
+
+  /**
+   * Description of the feature.
+   */
+  description: string | null;
+
+  /**
+   * The targeting rules for the feature.
+   */
+  isEnabled: {
```

Comment:
this is `targeting` in the API, but that didn't seem right so I've exposed this as `isEnabled`. Suggestions are welcome!

Reply:
`access`? And have the `isEnabled: true` inside maybe?

Reply:
I don't like access because we're increasingly heard that "access" doesn't really cover turning the thing on/off for companies. Leaning towards `toggleTargeting` and `configTargeting` atm..

Reply:
I approved and resolving this comment as it's being discussed internally.